packages/block-library/src/button/edit.js

```diff
@@ -41,9 +41,14 @@ const MAX_BORDER_RADIUS_VALUE = 50;
 const INITIAL_BORDER_RADIUS_POSITION = 5;
 
 function BorderPanel( { borderRadius = '', setAttributes } ) {
+	const initialBorderRadius = borderRadius;
 	const setBorderRadius = useCallback(
 		( newBorderRadius ) => {
-			setAttributes( { borderRadius: newBorderRadius } );
+			if ( newBorderRadius === undefined )
+				setAttributes( {
+					borderRadius: initialBorderRadius,
+				} );
+			else setAttributes( { borderRadius: newBorderRadius } );
```

Comment:
It's weird  that  this passes the linting checks, I thought `{}` were enforced.

Reply:
Oh, hmm 🤔

Reply:
Not a big deal, but if folks think it could be a good rule we can see if we should add it.